### PR TITLE
RED-368 preparations to work with lib

### PR DESCRIPTION
**Pull request by Ivan-dev-RED**

### CODE REVIEW

File: src/field/RedefinedDomainResolver.tsx [hunk 1]
```diff
@@ -3,7 +3,7 @@ import React from 'react';
 function RedefinedDomainResolver() {
   return (
     <div className="App">
-      <input />
+      <input placeholder="White yor domain" />
```
[the remark this answers is not in the document]
Comment (Ivan-dev-RED) [reply src/field/RedefinedDomainResolver.tsx:6]: Or Andrey)

File: package.json [hunk 2]
```diff
@@ -32,6 +35,8 @@
     "@types/react": "^18.0.27",
     "@types/react-dom": "^18.0.10",
     "react": "^18.2.0",
+    "react-app-rewire-alias": "^1.1.7",
+    "react-app-rewired": "^2.2.1",
     "react-dom": "^18.2.0",
     "react-scripts": "5.0.1",
     "typescript": "^4.9.5",
```
Comment (au-e2xlabs) [inline package.json:42]: all types, typescript and etc must be in dev dependencies or not?

Comment (Ivan-dev-RED) [reply package.json:42]: @au-e2xlabs 
May be)
If not difficult, could you change on your branch

Comment (au-e2xlabs) [reply package.json:42]: ok

**Review by au-e2xlabs — APPROVED** [review]

**Pull request merged**